lib/diagnostic.rb

```diff
@@ -5,22 +5,22 @@
 #
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
+# ruby lib/eample.rb
 
 Diagnostic.script = 'your response here'
```

Comment:
Make sure you put your answers where the diagnostic asks you to. I know you know this, but unfortunately you misspelled example :(